src/common/util.js

```diff
@@ -1,3 +1,7 @@
+function isRestrictedSymbol(symbol) {
+  return /^(BOOM|CRASH|STEP).+/i.test(symbol)
+}
```

Comment:
why not also write the filtered symbols function here and pass the active symbols to it